src/app/routes.js

```diff
@@ -76,6 +76,11 @@ export default (app, router, passport, io) => {
   analyticsRoutes(app, router, admin);
   pushRoutes(app, router);
 
+  // Provide a simple status page
+  router.get('/status', (req, res) => {
```

Comment:
Do you have to be logged in to request this? Is there a frontend page or are you supposed to check this with i.e. `curl`?